### ISSUE TITLE
Still having issues that Hashtags don't work

### ISSUE THREAD

**Issue by nthomasgsy** [cloudflare/wildebeest#345]
https://github.com/cloudflare/wildebeest/issues/243 closed by @xtuc 3 days ago, but please see following toot as evidence that this is not fixed:

https://nptcomputing.xyz/@nathan/a76689a7-fb09-407c-9fd9-ddd7f19eb8f4/

I am up to date with commits and look to be running v.98d0e1b.

Hashtags are not underlined in this post.  They are not clickable (a) on the instance, (b) in Mastodon Android App and (c) Pinafore.

Please could we get some evidence of testing before an issue is closed.

**Comment by xtuc** [comment]
Indeed, I closed the issue while only half of the work was done. The team is working on it.

**Comment by dario-piotrowicz** [comment]
@nthomasgsy I've merged a PR which should fix the hashtags not working, the PR was linked to resolve issue and thus it closed it, it should be alright, we'll double check if it all works fine with real data, if it doesn't not we'll reopen the issue 🙂

**Comment by nthomasgsy** [comment]
@dario-piotrowicz I have updated my instance and can confirm that Hashtags are now working.  Thanks!

**Comment by dario-piotrowicz** [comment]
@nthomasgsy that's awesome to hear! thanks a lot for confirming that it's working! 😃👍

**Comment by nthomasgsy** [comment]
@dario-piotrowicz a bit of further testing, let me know if you think either should be logged as an issue:

(1) While a hashtag from a post on my server works fine, if I go to open a hashtag on a post from a followed server it links to the hashtag listing on the remote server, so it's a page redirect if you click the link.  For example where the Raspberry Pi account has used #RaspberryPi it formulates a link to the hashtag listing on their server: https://raspberrypi.social/tags/RaspberryPi.  I see that on my Mastodon accounts the behaviour for hashtags in other people's posts is to redirect to the same hashtag on my local server, i.e. this would become https://mastodon.social/tags/RaspberryPi.  I think maybe if it Wildebeest did similar, i.e. https://mywildebeest.server/tags/RaspberryPi, then it wouldn't lead to a page redirect.  But then I also notice that when using Pinafore, it's substituting the localhost so we get https://pinafore.social/tags/RaspberryPi being used.

(2) When browsing my Wildebeest posts from a Mastodon server the hashtags are not being picked up at all - they're just plaintext with no hyperlink.  This means that it is not as useful to boost a post from a Wildebeest server, as the hashtags won't be interpreted.  So I end up dual posting as would probably other people.

Sorry to pick up on these, but thanks for making progress anyway!  Let me know if I need to try to explain better or provide you with links or screenshots.

**Comment by dario-piotrowicz** [comment]
@nthomasgsy sorry that some things are still not working as expected 😢, let's just reopen the issue

**Comment by dario-piotrowicz** [comment]
@nthomasgsy Since the issues you mentioned are related to other servers I need to have a check with @xtuc which is more familiar with that part.

regarding (2), could you confirm that you've tested that with new statuses created using your synched up instance?

**Comment by nthomasgsy** [comment]
Hi,
So my instance was updated yesterday with https://github.com/cloudflare/wildebeest/pull/347.

My example post on my instance with hashtags is https://nptcomputing.xyz/@nathan/eb4c2b46-6b7f-4e6c-988b-cc324ee968f4

That same post reposted from my TechHub mastodon account is at https://techhub.social/@nathan@nptcomputing.xyz/109950246835016830

Let me know if/when you want me to do a re-synchronisation and re-test.
